contracts/FantomAuction.sol

```diff
@@ -342,12 +342,12 @@ contract FantomAuction is OwnableUpgradeable, ReentrancyGuardUpgradeable {
     }
 
     /**
-     @notice Given a sender who has the highest bid on a item, allows them to withdraw their bid
+     @notice Allows the hightest bidder to withdraw the bid (after 12 hours) 
      @dev Only callable by the existing top bidder
      @param _nftAddress ERC 721 Address
      @param _tokenId Token ID of the item being auctioned
      */
-    function withdrawBid(address _nftAddress, uint256 _tokenId)
+    function transferBidOut(address _nftAddress, uint256 _tokenId)
```

Comment:
Can we just keep it as `withdrawBid` instead of renaming it? Otherwise, we ill have to unnecessarily update other code as well.

Reply:
Yeah we should do that.